commands/command_migrate_import.go

```diff
@@ -108,6 +122,20 @@ func migrateImportCommand(cmd *cobra.Command, args []string) {
 				return b, nil
 			}
 
+			if migrateFixup {
+				var ok bool
+				attrs := fixups.Applied(path)
+				for _, attr := range attrs {
+					if attr.K == "filter" && attr.V == "lfs" {
+						ok = !attr.Unspecified
```

Comment:
Note-to-self: I think that this should actually read:

```go
if attr.K == "filter" {
        ok = attr.V == "lfs" && !attr.Unspecified
}
```

What do you think?

Reply:
Hm, I think that we were mostly right, 😄. If `attr.Unspecified` holds, that means that we implicitly set `attr.V` to `""` (which happens [here][1], and is documented by Git [here][2]).

So, I think we can avoid checking `attr.Unspecified` here, since if it is true, we know that `attr.V != "lfs"`. I implemented that change in https://github.com/git-lfs/git-lfs/pull/3114/commits/853e7fce3b2b7209914466967c95fc6440ef1c2c, which holds up as I expected.

[1]: https://github.com/git-lfs/git-lfs/pull/3114/files#diff-1a27d1eb67e3578c4441409b111fc10cR102
[2]: https://git-scm.com/docs/gitattributes#gitattributes-Unspecified